.travis.yml

```diff
@@ -27,5 +27,5 @@ jobs:
     - name: "Command Tests: Unpaired Unlabeled"
       script:
         - deepreg_train -g "" --config_path config/unpaired_labeled_ddf.yaml --log_dir test
-        - deepreg_predict -g "" --ckpt_path logs/test/save/weights-epoch2.ckpt --mode test
+        - deepreg_predict -g "" --ckpt_path logs/test/save/weights-epoch2.ckpt --mode test -b 3
```

Comment:
what is -b? are you giving a fixed batch size? if this is required, this needs to go to docs

Reply:
this is not required, by default it's 1, the docs have it already https://deepreg.readthedocs.io/en/latest/docs/cli.html#id3

In this test, the test data only have 2 samples, which means using a batch_size = 3 would test the case of a not-full batch. Another option is to add one more sample data for testing and use batch_size 2.